docs/configuration.md

```diff
@@ -242,6 +242,20 @@ example of json file provided via METRICS_CONFIGFILE_CONTENTENVVAR or METRICS_CO
 |---------|---------------------|---------|----------|-----------|-------------|
 | config.env.tokenizers.json | CONFIG_ENV_TOKENIZERS_JSON | - | No | - | Preconfigured DIAL tokenizers list in JSON format |
 
+## Deployment Configuration
```

Comment:
add more details/context. it's unclear what is Deployment here.

Reply:
added